lib/av_client/aion_crypto.js

```diff
@@ -1076,38 +1076,35 @@ function addBigNums(bn1_hex, bn2_hex) {
   return sum_hex
 }
 
-module.exports = function() {
-  return {
-    'Curve': Curve,
-    'DiscreteLogarithmProof': DiscreteLogarithmProof,
-    'DiscreteLogarithmEqualityProof': DiscreteLogarithmEqualityProof,
-    'DiscreteLogarithmMultipleProof': DiscreteLogarithmMultipleProof,
-    'ElGamalScalarCryptogram': ElGamalScalarCryptogram,
-    'ElGamalPointCryptogram': ElGamalPointCryptogram,
-    'SchnorrSignature': SchnorrSignature,
-
-    'pointEquals': pointEquals,
-    'pointToBits': pointToBits,
-    'addPoints': addPoints,
-    'pointFromBits': pointFromBits,
-    'randomBN': randomBN,
-    'randomPoint': randomPoint,
-    'recoverYfromX': recoverYfromX,
-    'hashToBn': hashToBn,
-    'VOTE_ENCODING_TYPE': VOTE_ENCODING_TYPE,
-    'voteToPoint': voteToPoint,
-    'pointToVote': pointToVote,
-    'generateKeyPair': generateKeyPair,
-    'generateSchnorrSignature': generateSchnorrSignature,
-    'verifySchnorrSignature': verifySchnorrSignature,
-    'generateRandomNumber': generateRandomNumber,
-    'generateRandomPoint': generateRandomPoint,
-    'verifyEmptyCryptogramProof': verifyEmptyCryptogramProof,
-    'encryptVote': encryptVote,
-    'encryptVotePoint': encryptVotePoint,
-    'generateDiscreteLogarithmProof': generateDiscreteLogarithmProof,
-    'orderObjectByKeys': orderObjectByKeys,
-    'hashString': hashString,
-    'addBigNums': addBigNums,
-  }
+module.exports = {
+  'Curve': Curve,
```

Comment:
This should probably be changed to shorthand:

```suggestion
  Curve,
```

Reply:
Yes, great suggestion. Pushing it now.